recipes/enaml/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set version = "0.10.1" %}
+package:
+  name: enaml
+  version: {{ version }}
+
+source:
+  url: https://github.com/nucleic/enaml/archive/{{ version }}.tar.gz
```

Comment:
Please change the download URL to pypi.

Reply:
The package is not yet on PyPI, but it should be shortly I will update as soon as it is and let you know when it is done.